osrf/vrx
# Perception practice worlds

### How to test it?

Launch the simulation with the practice world:
```
ros2 launch vrx_gz competition.launch.py world:=practice_2022_perception0_task
```

Verify that the correct buoys show up at the right time according to each world file.
